### ISSUE TITLE
Reconcile Function context objects with DI

### ISSUE THREAD

**Issue by brettsam** [Azure/azure-webjobs-sdk#2314]
Things like `BindingContext`, `ExecutionContext`, etc, aren't registered with DI, but we may want to revisit this. Would allow you to build scoped services that would get these values injected with each function invocation.

**Comment by dalestone** [comment]
Yeah encountered the same issue, looks like it's scoped only to the actual functions, so you can't access it through normally DI start up unfortunately

**Comment by nachiappannk** [comment]
Has there been any updates in this?

Can I inject FunctionContext in to a scoped object now?

**Comment by gabbsmo** [comment]
Bump
